src/components/CalendarHeader.tsx

```diff
@@ -46,6 +48,7 @@ function _CalendarHeader<T>({
       <View style={[u['z-10'], u['w-50'], borderColor]} />
       {dateRange.map((date) => {
         const _isToday = isToday(date)
+        const _isActiveDate = activeDate ? isActiveDate(date, activeDate) : _isToday
```

Comment:
There should be performance improvement: ​if `activeDate` is passed, `_isToday` will be calculated but not used.

So, the logic should be like this:

```ts
       ​const shouldHighlight = activeDate ? date.isSame(activeDate, 'date') : isToday(date)
       ​return (
```

Also, please note that

- The var name `shouldHighlight` is more clear
- I think the function `isActiveDate` is a bit redundant, writing it inline must be simpler 
- `isSame(.., 'date')` is more clear than `isSame(.., 'day')` in this case

Sorry that I couldn't find it at the first time ;)

Reply:
Thank you for your review. I'll fix it!